masterlist.yaml

```diff
@@ -2455,6 +2455,7 @@ plugins:
           - lang: ja
             text: 'このModは人間種族（インペリアル、ブレトン、ノルド、レッドガード）を改変するModとは両立できません。また、バベットのfacegenデータ(0001D4B7.NIF)を含むModとは互換性がありません。'
   - name: 'Weapons & Armor Fixes_Remade.esp'
+    group: *underridesGroup
     after:
       - 'Guard Dialogue Overhaul.esp'
```

Comment:
It might also be worth adding GDO to underrides as well. That mod has not been updated in ages.